nixos/modules/services/networking/gokapi.nix

```diff
@@ -0,0 +1,136 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.services.gokapi;
+  settingsFormat = pkgs.formats.json { };
+  userSettingsFile = settingsFormat.generate "generated-config.json" cfg.settings;
+in
+{
+  options.services.gokapi = {
+    enable = lib.mkEnableOption "Lightweight selfhosted Firefox Send alternative without public upload. AWS S3 supported.";
+
+    mutableSettings =
+      (lib.mkEnableOption "Allow changes to the program config made by the program to persist between restarts. If disabled all required values must be set using nix, and all changes to config format over application updates must be resolved by user.")
+      // {
+        default = true;
+      };
```

Comment:
I think this should just use `mkOption` with `type = lib.types.bool` instead of `mkEnableOption`